src/fasta-recs.c

```
@@ -1,4 +1,35 @@
 #include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+// Reads a file and puts in a buffer, in an allocated memory space
+// https://stackoverflow.com/questions/3747086/reading-the-whole-text-file-into-a-char-array-in-c
+char* readFile(const char* filename){
+    
+    FILE *fp;
+    fp = fopen(filename, "r");
+    if( !fp ) perror("blah.txt"),exit(1);
```

That was a particularly weird way to use the comma operator. And "blah.txt"? What was that for? Also, wouldn't it be better to return an error than to crash the program with an exit?